### DISCUSSION TITLE
upgrade from old nextcloud container

### DISCUSSION THREAD

**Discussion by maldag** [nextcloud/all-in-one#5477] [Questions] [answered]
I'm trying to upgrade my nextcloud instance but this has gotten way too big.
Is there a way to just get the nextcloud image running without ssl or proxies on a just local network as I was using it before?

This one doesn't seem to run without ssl (which I cannot provide since running locally) (and self-signed certificates are possible but not really safe either) or is it time for me to move on to just sync my stuff?

**Comment by szaimen** [comment, marked as answer]
Hi, please ask this at https://help.nextcloud.com. Thanks!
